src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -721,12 +721,12 @@ bool SlaterDetBuilder::createMSDFast(MultiSlaterDeterminantFast* multiSD, xmlNod
       {
         //std::stringstream sstr;
         //sstr << "CIcoeff" << "_" << i;
-        #ifdef QMC_COMPLEX
+        //#ifdef QMC_COMPLEX
           // TODO: to be fixed by the complex VariableSet.
```

Comment:
Has this TODO been settled? If so, please clean up the commented code. Similar case below.

Reply:
Yes this TODO is done and the commented code has been deleted.